week2/homework/src/functions.js

```diff
@@ -0,0 +1,104 @@
+'use strict';
+
+const fs = require('fs');
+
+const functions = {
+  list: () => {
+    fs.readFile('./to-do.txt', 'utf8', (err, data) => {
+      if (err) {
+        console.log(err);
+      } else if (data === '') {
+        console.log('There is nothing in to-do list!');
+      } else {
+        let newData = data.split('\n');
+        newData.pop();
+        newData = newData.join('\n');
+        console.log(newData);
+      }
+    });
+  },
+
+  add: item => {
+    fs.appendFile('./to-do.txt', item + '\n', err => {
+      if (err) {
+        console.log(err);
+      } else {
+        console.log('New item added!');
+      }
+    });
+  },
+
+  remove: index => {
+    fs.readFile('./to-do.txt', 'utf8', (err, data) => {
+      if (err) {
+        console.log(err);
+      } else {
+        if (index > data.split('\n').length) {
```

Comment:
Well done! But what happens if I try to run it with negative numbers, for example:
`node index.js remove -100`

Reply:
Hello dear @saragerion I try to solve the minus problem and for this case, I think I solve. Thank you for your comments.